docs/upgrading.md

```diff
@@ -47,6 +47,12 @@ Custom metric names and labels must be valid Prometheus and OpenTelemetry names
 
 Custom metrics, as defined by a workflow, could be defined as one type (say counter) in one workflow, and then as a histogram of the same name in a different workflow. This would work in 3.5 if the first usage of the metric had reached TTL and been deleted. This will no-longer work in 3.6, and custom metrics may not be redefined. It doesn't really make sense to change a metric in this way, and the OpenTelemetry SDK prevents you from doing so.
 
+#### TLS
+
+The Prometheus `/metrics` endpoint now has TLS enabled by default.
+
+To disable this set `metricsConfig.secure` to `false`.
```

Comment:
```suggestion
To disable this set `metricsConfig.secure` to `false`.
```
nit: Following the prior upgrading notes from `--base-href`, no new line when listing how to use prior behavior. Saves some space.

Also TIL [no comma if <4 words in introductory preposition](https://www.grammarly.com/blog/commas-after-introductory-phrases/#:~:text=When%20an%20introductory%20prepositional%20phrase,four%20words%2C%20use%20a%20comma.)